config/samples/ccruntime/default/kustomization.yaml

```diff
@@ -36,5 +36,13 @@ patches:
     - op: add
       path: /spec/config/debug
       value: false
+    - op: add
+      path: /spec/config/environmentVariables
+      value:
+        # It means that we're relying on the cluster to already have
+        # containerd v1.7+ running.  If you know for sure that's not
+        # the case, please, set this variable to `true`
+        - name: "INSTALL_OFFICIAL_CONTAINERD"
+          value: "false"
```

Comment:
Related to my previous question on setting INSTALL_OFFICIAL_CONTAINERD to false in reqs-deploy.sh, if it makes sense to set the default there, and comment out this patch for users to uncomment and install containerd if needed ?

Reply:
Users can comment this out to install containerd if needed, IIRC.

Reply:
Ok. Would it make sense to document these variables ?
It can be done in a separate PR as well

Reply:
Yes, it would and I will do it in a separate PR before the release.
This should also be mentioned in the release notes,